src/schema_compile.c

```diff
@@ -1761,7 +1758,6 @@ lys_compile(struct lys_module *mod, uint32_t options, ly_bool recompile, struct
     LOG_LOCBACK(0, 0, 1, 0);
     lys_compile_unres_mod_erase(&ctx, ret);
     if (ret) {
-        lys_precompile_augments_deviations_revert(ctx.ctx, mod);
```

Comment:
how is this related to the commit f2a22c6?

Reply:
The message was not clear and now it should be better. I suppose it could have been a separate commit but the reasoning is the same, it is not needed to call this function here. Also, looking at it now, it is irrelevant because augments and deviations are not applied in `lys_compile()` anymore.